src/web/demo/src/main/java/org/geoserver/web/demo/MapPreviewPage.java

```diff
@@ -218,7 +218,7 @@ private Component buildJSWMSSelect(
         RepeatingView wmsFormats = new RepeatingView("wmsFormats");
         for (int i = 0; i < wmsOutputFormats.size(); i++) {
             String wmsOutputFormat = wmsOutputFormats.get(i);
-            String label = translateFormat("format.wms.", wmsOutputFormat);
+            String label = translateFormat( "format.wms.", wmsOutputFormat );
```

Comment:
To experiment with skipping un translated output formats use identity check:

```
if( label == wmsOutputFormat) continue;  // Skip non localized output format
```

However this is unfair to community modules developing output formats,  as they are unable to update GeoServerApplication.properties